.flake8

```diff
@@ -0,0 +1,5 @@
+[flake8]
```

Comment:
Is this `ini` file format? If so, is it possible to leave a comment saying something like:

```
Configure flake8 to be compatible with black: https://ljvmiranda921.github.io/notebook/2018/06/21/precommits-using-black-and-flake8/
```